ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml

```diff
@@ -16,6 +17,19 @@
       - mimic
       - luminous
       - nautilus
+    exclude:
+      - ceph-branch: master
+        distro: centos7
+      - ceph-branch: nautilus
+        distro: centos7
```

Comment:
I think you want to exclude centos 8 for nautilus and not centos 7 right ?

Reply:
Hmm I intended to skip centos7 originally, since the nautilus nightlies were build with ceph-ansible master. But I have since changed that to ceph-ansible 4.0.

That hasn't fixed the nightlies however. Looking into it.

Reply:
so you're right, nautilus will be tested with centos7. I have a few fixes that get the nautilus tets to run again.